operator/hack/addons_ocp.yaml

```diff
@@ -118,7 +118,7 @@ metadata:
 data:
   promtail.yaml: |
     clients:
-      - url: http://lokistack-dev-gateway-http.openshift-logging.svc:8080/api/logs/v1/application/loki/api/v1/push
+      - url: https://lokistack-dev-gateway-http.openshift-logging.svc:8080/api/logs/v1/application/loki/api/v1/push
```

Comment:
This probably does not work without:
```
# If connecting to a TLS server, configures how the TLS
# authentication handshake will operate.
tls_config:
  # The CA file to use to verify the server
  [ca_file: <string>]
```
See: https://grafana.com/docs/loki/latest/clients/promtail/configuration/#clients

Reply:
done